### PR TITLE
update for 5.0

### PR DESCRIPTION
**Pull request by jlstrater**
- add Kotlin DSL side by side example
- show that the init plugin now uses a package name instead of default package
- remove the unnecessary and confusing text about the init plugin
- bump the minimum java version to 8

### CODE REVIEW

File: samples/output/build.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+> Task :clean
```
Comment (jlstrater) [inline samples/output/build.txt]: 🙈remove this. This isn't part of the build command.

File: samples/groovy-dsl/code/build.gradle [hunk 2]
```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'java'
+
```
Comment (eskatos) [inline samples/groovy-dsl/code/build.gradle]: :o: Nitpicking: comments added by the init plugin were removed, I'm fine with that. This extra blank line could be removed.

File: samples/kotlin-dsl/code/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+plugins {
+    java
+
```
Comment (eskatos) [inline samples/kotlin-dsl/code/build.gradle.kts]: :o: Nitpicking: comments added by the init plugin were removed, I'm fine with that. This extra blank line could be removed.

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-rc-2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-rc-1-bin.zip
```
Comment (eskatos) [inline gradle/wrapper/gradle-wrapper.properties]: :x: should use the latest available 5.0-rc-3 at the time of writing

File: contents/index.adoc [hunk 5]
```diff
@@ -16,45 +16,41 @@ Gradle comes with a built-in plugin called the Build Init plugin. It is document
...
+Project name (default: demo):
+Source package (default: demo):
 
 BUILD SUCCESSFUL
 ----
 
-The `init` task runs the `wrapper` task first, which generates the `gradlew` and `gradlew.bat` wrapper scripts. Then it creates the new project with the following structure:
+The `init` task runs the `wrapper` task first, which generates the `gradlew` and `gradlew.bat` wrapper scripts. Then it creates the new project with the following structure (when using the Groovy DSL):
 
 [listing]
```
Comment (eskatos) [inline contents/index.adoc]: :x: the listing should show the layout for groovy and kotlin DSLs

Comment (jlstrater) [reply contents/index.adoc]: I added both project layouts one after the other. As far as I can tell there isn't a way to do this as tabs. @eriwen the tabs only work for includes from other files, right?

Comment (eskatos) [reply contents/index.adoc]: @jlstrater, there's https://guides.gradle.org/creating-new-gradle-builds/#initialize_a_project that does it, see https://github.com/gradle-guides/creating-new-gradle-builds/blame/master/contents/index.adoc#L37-L46

Comment (jlstrater) [reply contents/index.adoc]: ah right. I can pull it into a separate output file.

**Pull request merged**